app/javascript/components/data-tables/db-list/index.jsx

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { tableData, onSelectRender } from './helper';
 import MiqDataTable from '../../miq-data-table';
+import NoRecordsFound from '../../no-recods-found';
```

Comment:
```suggestion
import NoRecordsFound from '../../no-records-found';
```

Is this not a typo?

Reply:
The original file name has a typo in it

Reply:
Ah ok - we should get that cleaned up in a follow-up

Reply:
Raised a PR for the typo correction